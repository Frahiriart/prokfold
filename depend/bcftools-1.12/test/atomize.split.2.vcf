##fileformat=VCFv4.3
##contig=<ID=1,length=135006516>
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	sample
1	100	.	CC	GG	.	.	.	GT	0/1
1	110	.	C	T,<*>	.	.	.	GT	0/1
