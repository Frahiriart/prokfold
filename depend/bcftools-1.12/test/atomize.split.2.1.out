##fileformat=VCFv4.3
##FILTER=<ID=PASS,Description="All filters passed">
##contig=<ID=1,length=135006516>
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##INFO=<ID=OLD_REC,Number=1,Type=String,Description="Original variant. Format: CHR|POS|REF|ALT|USED_ALT_IDX">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	sample
1	100	.	C	G	.	.	OLD_REC=1|100|CC|GG|1	GT	0/1
1	101	.	C	G	.	.	OLD_REC=1|100|CC|GG|1	GT	0/1
1	110	.	C	T,<*>	.	.	.	GT	0/1
